x-pack/plugins/infra/public/index.scss

```diff
@@ -3,22 +3,13 @@
 
 /* Infra plugin styles */
 
-// Prefix all styles with "inf" to avoid conflicts.
-// Examples
-//   infChart
-//   infChart__legend
-//   infChart__legend--small
-//   infChart__legend-isLoading
-
-.infReactRoot {
+.infra-container-element {
   background-color: $euiColorEmptyShade;
   position: absolute;
   top: 0;
   left: 0;
   bottom: 0;
   right: 0;
-  // display: flex;
-  // flex-direction: column;
   align-items: flex-start;
```

Comment:
I haven't run this locally, but at a quick glance, the `align-items`, `flex`, and `flex-direction` css prosperities look unnecessary if we're removing the `display: flex`. Can you confirm that they are needed?

Reply:
I’ve removed these commented out ones, but the `display: flex` and `flex-direction: column` rules still exist further down.

Reply:
Ah perfect, my mistake :). LGTM